test/blackbox-tests/test-cases/lib-collision/lib-collision-public-same-public-name.t

```diff
@@ -23,12 +23,12 @@ different folders.
 Without any consumers of the libraries
 
   $ dune build
```

Comment:
this error becomes better: we now use the `loc` for the `(public_name ..)` field, which is the actual conflict.